### PR TITLE
add alert command

### PR DESCRIPTION
**Pull request by leo-chartier**
## Summary

Add an `alert` command, similar to `reminder` except it takes the final date rather than the duration.

## Checklist

- [x] If code changes were made then they have been tested.
    - [ ] I have updated the documentation to reflect the changes.
- [ ] This PR fixes an issue.
- [x] This PR adds something new (e.g. new method or parameters or reference to an issue).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

File: cogs/useful.py [hunk 1]
```diff
@@ -46,6 +55,39 @@ async def reminder(self, interaction: discord.Interaction, hours: int, minutes:
 		await asyncio.sleep(seconds+minutes*60+hours*(60**2))
 		await interaction.channel.send(f":bell: <@{interaction.user.id}> Reminder (<t:{remind_in}:R>): {message}")
 
+	@bot_has_permissions(send_messages=True)
+	@app_commands.command(name="alarm", description="Reminds you of something.")
+	@app_commands.describe(month="Month.", day="Day.", hour="Hour.", minute="Minute.", second="Second.", message="Your alarm message.")
+	@app_commands.choices(month=[app_commands.Choice(name=m, value=i+1) for i, m in enumerate(_MONTHS)])
+	async def alert(self, interaction: discord.Interaction, month: int=-1, day: int=-1, hour: int=9, minute: int=0, second: int=0, message: str=None) -> None:
```
Comment (PaulMarisOUMary) [inline cogs/useful.py:62]:
```suggestion
	async def alert(self, interaction: discord.Interaction, message: str, month: int=-1, day: int=-1, hour: int=9, minute: int=0, second: int=0):
```

**Review by PaulMarisOUMary — CHANGES_REQUESTED** [review]
Thanks for your contribution !
It could be simpler for a user to make the `message` parameter and at least one *time* parameter must be required.

File: cogs/useful.py [hunk 2]
```diff
@@ -46,6 +55,39 @@ async def reminder(self, interaction: discord.Interaction, hours: int, minutes:
...
+			day = now.day
+		
+		try:
+			due = datetime(year=now.year, month=month, day=day, hour=hour, minute=minute, second=second)
+		except ValueError:
+			await interaction.response.send_message("You must specify a valid date and time")
+			return
+
+		dt = due - now
+		if dt < 0:
```
[the remark this answers is not in the document]
Comment (leo-chartier) [reply cogs/useful.py]: Fixed by previous commit

**Review by PaulMarisOUMary — APPROVED** [review]
LGTM 👍

**Pull request merged**